libraries/AP_Follow/AP_Follow.cpp

```diff
@@ -134,419 +155,634 @@ const AP_Param::GroupInfo AP_Follow::var_info[] = {
     AP_GROUPINFO("_ALT_TYPE", 10, AP_Follow, _alt_type, AP_FOLLOW_ALT_TYPE_DEFAULT),
 #endif
 
+// May add option to turn on input shaping, turn on acceleration due to rotation of offset
+
     // @Param: _OPTIONS
     // @DisplayName: Follow options
     // @Description: Follow options bitmask
     // @Values: 0:None,1: Mount Follows lead vehicle on mode enter
     // @User: Standard
     AP_GROUPINFO("_OPTIONS", 11, AP_Follow, _options, 0),
 
+    // @Param: _ACCEL_NE
+    // @DisplayName: Acceleration limit for the horizontal kinematic input shaping
+    // @Description: Acceleration limit of the horizontal kinematic path generation used to determine how quickly the ship varies in velocity
```

Comment:
```suggestion
    // @Description: Acceleration limit of the horizontal kinematic path generation used to determine how quickly the target varies in velocity
```
Not everyone is landing on ships ;)